core/logger/errors.ts

```diff
@@ -14,16 +14,24 @@ export function UNKNOWN_FIELD(name: string): string {
   );
 }
 
-export function UNKNOWN_ACTION(): string {
+export function UNKNOWN_ACTION(code: string, pos: number): string {
+  const nextLineIndex = code.indexOf('\n', pos);
```

Comment:
I think it would be possible to change `\n` to `\n\r` and then we would go back to `CRLF` instead of `LF` in our schemas. It is not critical, just another thing to think about.  Using `LF` is still a good solution